usr/lib/sticky/sticky.py

```diff
@@ -9,16 +15,8 @@
 gi.require_version('Gspell', '1')
 gi.require_version('Gtk', '3.0')
 gi.require_version('XApp', '1.0')
```

Comment:
These require_version calls *MUST* occur before the actual imports of those modules